includes/login-register.php

```diff
@@ -159,6 +159,62 @@ function edd_log_user_in( $user_id, $user_login, $user_pass, $remember = false )
 
 }
 
+add_filter( 'wp_login_errors', 'edd_login_register_error_message', 10, 2 );
+/**
+ * Changes the WordPress login confirmation message when using EDD's reset password link.
+ *
+ * @since 2.10
+ * @param object \WP_Error $errors
+ * @param string $redirect
+ * @return void
+ */
+function edd_login_register_error_message( $errors, $redirect ) {
+	$action_is_confirm   = 'confirm' === filter_input( INPUT_GET, 'checkemail', FILTER_SANITIZE_STRING );
+	$edd_action_is_reset = 'confirm' === filter_input( INPUT_GET, 'edd_reset_password', FILTER_SANITIZE_STRING );
+	$redirect_url        = filter_input( INPUT_GET, 'edd_redirect', FILTER_SANITIZE_URL );
+	if ( $action_is_confirm && $edd_action_is_reset ) {
```

Comment:
```suggestion
	if ( $action_is_confirm && $edd_action_is_reset && $redirect_url ) {
```

No point customizing the error message if we don't have a redirect URL for whatever reason.